src/main/java/frc/team2412/robot/subsystem/IndexSubsystem.java

```diff
@@ -26,7 +26,8 @@
 
         // Index Motor Speeds
 
-        public static double INDEX_IN_SPEED = 0.2; // will change values later
+        public static double INDEX_FEEDER_SPEED = 0.25;
+        public static double INDEX_IN_SPEED = 0.35; // will change values later
```

Comment:
Let's finally remove the "will change later" comments.  We believe we are pretty much tuned now.